### PR TITLE
Feature/story initials

### PR DESCRIPTION
**Pull request by negreirosleo**
It adds the owner initials to the StoryInfo

### CODE REVIEW

File: app/assets/javascripts/components/story/StoryItem.js [hunk 1]
```diff
@@ -104,12 +104,22 @@ const StoryLabels = ({ labels }) => {
   );
 };
 
-const StoryInfo = ({ storyType ,title, labels, estimate }) => (
+export const StoryTitle = ({title, ownedByInitials, ownedByName}) => (
```
Comment (brunohkbx) [inline app/assets/javascripts/components/story/StoryItem.js]: You should keep the space inside of curly braces
```suggestion
export const StoryTitle = ({ title, ownedByInitials, ownedByName }) => (
```

File: spec/javascripts/components/story/story_title_spec.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+      ownedByInitials: "FB",
+      ownedByName: "Foo Bar"
+    };
+    const wrapper = shallow(<StoryTitle {...props} />);
+
+    expect(wrapper.find(`.Story__title`).text()).toContain(`${props.title}`);
+    expect(wrapper.find(`.Story__title`).text()).toContain(`${props.ownedByInitials}`);
+    expect(wrapper.find(`.Story__initials`).props().title).toEqual(`${props.ownedByName}`);
+  });
+});
```
Comment (brunohkbx) [inline spec/javascripts/components/story/story_title_spec.js]: This file is missing the EOF

File: app/assets/javascripts/components/story/StoryItem.js [hunk 3]
```diff
@@ -104,12 +104,22 @@ const StoryLabels = ({ labels }) => {
...
+  <div className="Story__title">
+    {title} <abbr className="Story__initials" title={`${ownedByName}`}>{ownedByInitials}</abbr>
```
Comment (gutofoletto) [inline app/assets/javascripts/components/story/StoryItem.js]: can we break the line here to place `abbr` on another line?

Comment (negreirosleo) [reply app/assets/javascripts/components/story/StoryItem.js]: sure we can

**Pull request merged**